beacon-chain/cache/depositcache/deposits_cache_test.go

```diff
@@ -409,6 +409,16 @@ func TestFinalizedDeposits_UtilizesPreviouslyCachedDeposits(t *testing.T) {
 	}
 }
 
+func TestFinalizedDeposits_DepositsAreNil_ReturnsNil(t *testing.T) {
+	dc := DepositCache{}
+
+	deposits := dc.finalizedDeposits
+
+	if deposits != nil {
```

Comment:
Can we use the new `assert` package? 

Reference: #6597

Reply:
Not really because the package only has a `NotNil` condition and I want the variable to be `nil`

Reply:
You can use `Equal` to compare against nil

Reply:
`deposits_cache_test.go:418 Deposits should be nil, got: <nil>, want: <nil>`

I guess it's a similar problem as with https://github.com/prysmaticlabs/prysm/issues/6590#issuecomment-658417687